buildSrc/src/main/java/Dependencies.kt

```diff
@@ -149,10 +92,14 @@ object Dependencies {
  * ```
  */
 @JvmName("get")
-fun getDependencyFromGroovy(path: String): String = Dependencies.resolveObject(
-    path.toLowerCase(US)
-        .split(".")
-)
+fun getDependencyFromGroovy(path: String): String = try {
+  Dependencies.resolveObject(
+      path.toLowerCase(US)
+          .split(".")
+  )
+} catch (e: Throwable) {
+  throw IllegalArgumentException("Error resolving dependency: $path", e)
```

Comment:
?

Reply:
Better than the "no element matches predicate" exception that otherwise gets thrown if a groovy gradle file tries resolving a non-existent dependency.